msteams-platform/m365-apps/overview.md

```diff
@@ -34,7 +34,7 @@ Reach your users where they are, right in the context of their work by extending
 
 :::image type="content" source="images/outlook-office-teams-personal-tab.png" alt-text="The screenshot is an example that shows Personal tab running in Outlook, Microsoft 365, and Teams.":::
 
-On mobile, you can test and debug your Teams personal tab running on [Microsoft 365 app for Android](extend-m365-teams-personal-tab.md#microsoft-365-app-for-android).
+On mobile, you can test and debug your Teams personal tab running on [Microsoft 365 app for Android](extend-m365-teams-personal-tab.md#microsoft-365-app-for-mobile-(android)).
```

Comment:
Probably need to redo this link depending on if/how you change section titles as suggested above